market_maker_keeper/erisx_market_maker_keeper.py

```diff
@@ -264,8 +264,7 @@ def __init__(self, args: list):
 
         self.arguments = parser.parse_args(args)
 
-        logging.basicConfig(format='%(asctime)-15s %(levelname)-8s %(message)s',
-                            level=(logging.DEBUG if self.arguments.debug else logging.INFO))
+        setup_logging(self.arguments)
```

Comment:
this is normally handled in the initalization of `CEXKeeperAPI`, but believe this needs to be moved up ahead of the PyEx client instantiation as there are instantiation issues prior to logging setup

Reply:
This is just a refactoring though; the manual call to `logging.basicConfig` was already being called before `ErisxApi` was constructed.

Reply:
Yeah, good point. I guess the idea was that, beyond cleaning things up, this could help by ensuring the urllib3 connection pooling logic was being used, but on closer inspection that's probably unrelated to the logging issue as we aren't using http

Reply:
[This line](https://github.com/makerdao/market-maker-keeper/blob/master/market_maker_keeper/util.py#L26) is there merely to reduce logspew.

Reply:
I see. Do you have any thoughts on other mechanisms that could be causing logs to not be generated on startup?

Reply:
Actually, logs may be generated but it could be a logging level issue as the initial handshake levels are all `DEBUG`